### PR TITLE
minor type in readme

### PR DESCRIPTION
**Pull request by julzhk**
Otherwise: spelling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,5 +191,5 @@ python manage.py rendervariations 'app_name.model_name.field_name' [--replace] [
 ```
 The `replace` option will replace all existing files.
 The `ignore-missing` option will suspend missing source file errors and keep
-rendering variations for other files. Othervise command will stop on first
+rendering variations for other files. Otherwise command will stop on first
```
Comment (codingjoe) [inline README.md:194]:
```suggestion
rendering variations for other files. Otherwise, the command will stop on first
```

**Review by codingjoe — COMMENTED** [review]
Thanks, @julzhk,

I love a good pull-request to enhance the documentation. Since we're at it, I had one small suggestion.

Best
Joe!

File: README.md (hunk 1 above)
Comment (julzhk) [reply README.md]: Yep - that's better - words is hard. lol

File: README.md [hunk 2]
```diff
@@ -4,19 +4,29 @@
 
 # Django Standardized Image Field
 
-Django Field that implement the following features:
+## Why would I want this?
 
-* Django-Storages compatible (S3)
-* Resize images to different sizes
+This is a drop-in replacement for the [Django ImageField](https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.ImageField) that provides a standardized way to handle image uploads.
```
Comment (codingjoe) [inline README.md:9]:
```suggestion
This is a drop-in replacement for the Django's [ImageField](https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.ImageField) that provides a standardized way to handle image uploads.
```

File: README.md [hunk 3]
```diff
@@ -4,19 +4,29 @@
...
+* Resizes images to different sizes
 * Access thumbnails on model level, no template tags required
-* Preserves original image
-* Asynchronous rendering (Celery & Co)
-* Restrict accepted image dimensions
-* Rename files to a standardized name (using a callable upload_to)
+* Preserves original images
+* Can be rendered asynchronously (ie as a [Celery job](https://realpython.com/asynchronous-tasks-with-django-and-celery/))
+* Restricts acceptable image dimensions
+* Renames file to a standardized name format (using a callable `upload_to` function, see below)
```
Comment (codingjoe) [inline README.md:25]:
```suggestion
* Renames a file to a standardized name format (using a callable `upload_to` function, see below)
```

File: README.md [hunk 4]
```diff
@@ -4,19 +4,29 @@
...
 
 ## Installation
 
-Simply install the latest stable package using the command
+Simply install the latest stable package using the following command
```
Comment (codingjoe) [inline README.md:29]:
```suggestion
Simply install the latest stable package using the following command:
```

File: README.md [hunk 5]
```diff
@@ -28,11 +38,13 @@ and add `'stdimage'` to `INSTALLED_APP`s in your settings.py, that's it!
...
 
+Now it's instally you can use either: `StdImageField` or `JPEGField`.
+
 `StdImageField` works just like Django's own
 [ImageField](https://docs.djangoproject.com/en/dev/ref/models/fields/#imagefield)
-except that you can specify different sized variations.
+except that you can specify different size variations.
 
-The `JPEGField` works similar to the `StdImageField` but all size variations are
+The `JPEGField` is the same as the `StdImageField` but all images are
```
Comment (codingjoe) [inline README.md:47]:
```suggestion
The `JPEGField` is identical to the `StdImageField` but all images are
```

**Review by codingjoe — COMMENTED** [review]
Oh wow, thanks. I had some very small suggestions. Again, thanks for putting so much work into this. I really appreciate it!

File: README.md (hunk 2 above)
Comment (julzhk) [reply README.md:9]: 'for the Django's'... shouldn't have 'the' I believe..

**Comment by julzhk** [comment]
I'm very glad you like some of my suggested edits!
& I really appreciate the code so happy to be able to contribute in some small way.

File: README.md (hunk 2 above)
Comment (codingjoe) [reply README.md:9]: Oh yes, certainly. Good spot.

**Review by codingjoe — APPROVED** [review]
🎉

**Pull request merged**